app/routes/publications/publication.js

```diff
@@ -9,8 +9,7 @@ export default class PublicationRoute extends Route {
     return this.store.queryOne('publication-flow', {
       'filter[:id:]': params.publication_id,
       include: [
-        'case',
-        'case.subcases',
+        'case.decisionmaking-flow.subcases',
```

Comment:
Is this include still required here?  It used to be here for the check executed in `getIsViaCouncilOfMinisters()` in the publication-service, but it seems to me the subcases of the decisionmaking-flow are no longer needed in this route.

Reply:
:thought_balloon: might be missing something but it seems they were never used? `getIsViaCouncilOfMinisters` doesn't use the subcases either, only the `publicationFlow`'s `decisionActivity`. And grepping for `\.subcases` in the codebase doesn't turn up results for the publication routes/components  :thinking:

Reply:
`getIsViaCouncilOfMinisters()` previously contained logic based on subcases. So that include could actually already have been removed before.